### PR TITLE
[zh-cn]: update the translation of TypedArray prototype `length` property

### PR DESCRIPTION
**Pull request by T34-active**
### Description
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/length
### Related issues and pull requests
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/typedarray/length/index.md
* Last commit: https://github.com/mdn/content/commit/c2445ce1dc3a0170e2fbfdbee10e18a7455c2282

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/typedarray/length/index.md [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
 title: TypedArray.prototype.length
 slug: Web/JavaScript/Reference/Global_Objects/TypedArray/length
+l10n:
+  sourceCommit: c2445ce1dc3a0170e2fbfdbee10e18a7455c2282
 ---
 
 {{JSRef}}
 
-**`length`** 访问器属性表示类型化数组的长度（元素数）。
+{{jsxref("TypedArray")}} 实例的 **`length`** 访问器属性返回该类型数组的（以元素为单位）长度。
```
Comment (T34-active) [inline files/zh-cn/web/javascript/reference/global_objects/typedarray/length/index.md:10]:
```suggestion
{{jsxref("TypedArray")}} 实例的 **`length`** 访问器属性返回该类型化数组的（以元素为单位）长度。
```

File: files/zh-cn/web/javascript/reference/global_objects/typedarray/length/index.md [hunk 2]
```diff
@@ -1,33 +1,35 @@
...
 title: TypedArray.prototype.length
 slug: Web/JavaScript/Reference/Global_Objects/TypedArray/length
+l10n:
+  sourceCommit: c2445ce1dc3a0170e2fbfdbee10e18a7455c2282
 ---
 
 {{JSRef}}
 
-**`length`** 访问器属性表示类型化数组的长度（元素数）。
+{{jsxref("TypedArray")}} 实例的 **`length`** 访问器属性返回该类型化数组的（以元素为单位）长度。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/typedarray/length/index.md:10]:
```suggestion
{{jsxref("TypedArray")}} 实例的 **`length`** 访问器属性返回该类型化数组的长度（以元素为单位）。
```

File: files/zh-cn/web/javascript/reference/global_objects/typedarray/length/index.md [hunk 3]
```diff
@@ -1,33 +1,35 @@
...
 
 {{EmbedInteractiveExample("pages/js/typedarray-length.html","shorter")}}
 
 ## 描述
 
-`length` 是一个访问器属性，它的 set 访问器函数是`undefined`，意思是你只能够读取这个属性。它的值在*TypedArray*构造时建立，不能被修改。如果 _TypedArray_ 没有指定`byteOffset` 或者 `length`，会返回所引用的`ArrayBuffer` 的`length`。_TypedArray_ 是这里的 [TypedArray 对象](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/TypedArray#TypedArray_objects)之一。
+`length` 属性是一个设置访问器函数为 `undefined` 的访问器属性，这意味着你只能读取该属性。该值在构建 _TypedArray_ 时确定，不可更改。如果 _TypedArray_ 没有指定 `byteOffset` 或 `length`，则将返回引用的 {{jsxref("ArrayBuffer")}} 的长度。_TypedArray_ 是 [TypedArray 对象](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/TypedArray#typedarray_对象)之一。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/typedarray/length/index.md:16]: setter/getter 相关倾向于不翻译

```suggestion
`length` 属性是一个 set 访问器函数为 `undefined` 的访问器属性，这意味着你只能读取该属性。该值在构建 _TypedArray_ 时确定，不可更改。如果 _TypedArray_ 没有指定 `byteOffset` 或 `length`，则将返回引用的 {{jsxref("ArrayBuffer")}} 的长度。_TypedArray_ 是 [TypedArray 对象](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/TypedArray#typedarray_对象)之一。
```

File: files/zh-cn/web/javascript/reference/global_objects/typedarray/length/index.md [hunk 4]
```diff
@@ -1,33 +1,35 @@
...
 
-var uint8 = new Uint8Array(buffer);
-uint8.length; // 8 (符合 buffer 的 length)
+let uint8 = new Uint8Array(buffer);
+uint8.length; // 8（与缓冲区 length 相匹配）
 
-var uint8 = new Uint8Array(buffer, 1, 5);
-uint8.length; // 5 (在 Uint8Array 构造时指定)
+uint8 = new Uint8Array(buffer, 1, 5);
+uint8.length; // 5（在构建 Uint8Array 时指定）
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/typedarray/length/index.md:29]:
```suggestion
uint8.length; // 5（在构造 Uint8Array 时指定）
```

**Review by jasonren0403 — COMMENTED** [review]
some minor changes

**Review by jasonren0403 — APPROVED** [review]

**Pull request merged**